Components/TaskList.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { FlatList } from 'react-native';
+import Task from 'ToDoProject/Components/Task'
+
+export default TaskList = (props) => {
+    return (
+        <FlatList
+            data={props.tasks}
+            extraData={props.extraData}
+            renderItem={({ item }) => <Task onDelete={props.onDelete} taskNumber={item.key} text={item.description}></Task>}
```

Comment:
get index from item and pass to props of task
`<Task index={item.index} onDelete={props.onDelete} taskNumber={item.key} text={item.description}></Task>`